### PR TITLE
data: updates

### PR DESCRIPTION
**Pull request by mikejw**

### CODE REVIEW

File: data/mikejw.json [hunk 1]
```diff
@@ -2,22 +2,87 @@
...
+    },
+    {
+      "title": "First Lead ReactJS Developer Contract Role",
+      "icon": "FaReact",
+      "date": "September 2017"
+    },
+    {
+      "title": "First DevOps Engineer (CI/CD, IAC) Contract Role",
+      "icon": "SiTerraform",
+      "date": "November 2018",
```
Comment (shubhsharma19) [inline data/mikejw.json]: Remove the comma here

**Review by shubhsharma19 — APPROVED** [review]
LGTM 💯

File: data/mikejw.json (hunk 1 above)
Comment (mikejw) [reply data/mikejw.json]: Already done :)

**Review by khushalarora — APPROVED** [review]
Looks good  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**